docs/src/v1-api/consuming/override.md

```diff
@@ -0,0 +1,198 @@
+# Consume and modify dream2nix packages
+
+## Given the following package
+`upstream/my-package.nix`
+```nix
+{config, lib, dream2nix, ...}: {
+
+  imports = [
+    dream2nix.modules.nodejs.mkDerivation
+    dream2nix.modules.nodejs.package-lock
+  ];
+
+  pname = "my-package";
```

Comment:
`pname` is one of those historical accidents. Maybe it's a good opportunity to rename it to `name`.

Reply:
I considered that, but am not sure if this would be a good idea as of now. We still use `nixpkgs.mkDerivation` as a builder for most derivations. If we put an adapter in front of it that changes the semantics we also have to ensure that this change is reflected by all downward compatibility mechanisms. For example we need to patch `overrideAttrs` to have that behavior as well.

For keeping things simple we might want to keep drv-parts based modules as close as possible to being a drop in replacement for the underlying builder API.

Reply:
This was resolved in the mean time. There is no pname anymore.